operate/qa/backup-restore-tests/src/test/java/io/camunda/operate/qa/backup/DataGenerator.java

```diff
@@ -63,12 +63,12 @@ public class DataGenerator {
       DateTimeFormatter.ofPattern("yyyy-MM-dd").withZone(ZoneId.systemDefault());
 
   /**
-   * ZeebeClient must not be reused between different test fixtures, as this may be different
+   * CamundaClient must not be reused between different test fixtures, as this may be different
    * versions of client in the future.
    */
   private ZeebeClient zeebeClient;
```

Comment:
❓ So, should we rather move to the CamundaClient then?

Reply:
Nope, I will rollback the changes in the doc. I think operate devs should decide if they want to switch and using camunda client